src/DashboardUI/src/pages/application/components/ApplicationSubmissionFormLayout.tsx

```diff
@@ -0,0 +1,555 @@
+import { createRef, useRef, useState } from 'react';
+import Button from 'react-bootstrap/esm/Button';
+import Form from 'react-bootstrap/esm/Form';
+import InputGroup from 'react-bootstrap/esm/InputGroup';
+import { useNavigate } from 'react-router-dom';
+import { NotImplementedPlaceholder } from '../../../components/NotImplementedAlert';
+import { states } from '../../../config/states';
+import { useConservationApplicationContext } from '../../../contexts/ConservationApplicationProvider';
+import { ApplicationSubmissionForm } from '../../../data-contracts/ApplicationSubmissionForm';
+import {
+  CompensationRateUnits,
+  CompensationRateUnitsLabelsPlural,
+  CompensationRateUnitsLabelsSingular,
+  CompensationRateUnitsOptions,
+} from '../../../data-contracts/CompensationRateUnits';
+import { formatNumber } from '../../../utilities/valueFormatters';
+import ApplicationFormSection from '../components/ApplicationFormSection';
+import { ApplicationDocumentUpload } from '../create/ApplicationDocumentUpload';
+const responsiveOneQuarterWidthDefault = 'col-lg-3 col-md-4 col-12';
+const responsiveOneThirdWidthDefault = 'col-lg-4 col-md-6 col-12';
+const responsiveHalfWidthDefault = 'col-lg-6 col-12';
+
+function ApplicationSubmissionFormLayout() {
```

Comment:
Drop the `Layout` and strip it down to just the form itself is what I'm thinking
```suggestion
function ApplicationSubmissionForm() {
```